src/tv2_afvd_showstyle/__tests__/configs.ts

```diff
@@ -12,15 +12,15 @@ function getSisyfosLayers(configName: string, id: string): string[] {
 		case 'SourcesFeed':
 			return ['sisyfos_source_live_' + id]
 		case 'SourcesDelayedPlayback':
-			return ['sisyfos_source_' + id.toLowerCase().replace(' ', '_')]
+			return ['sisyfos_source_' + id.toLowerCase().replace(/ /g, '_')]
 	}
 
 	return []
 }
 
 // TODO: Broken
 function prepareConfig(
-	conf: string,
+	conf: Array<{ id: string; switcherSource: number; sisyfosLayers?: string[] }>,
```

Comment:
I don't remember if we use `Array<T>` as the primary array type syntax in old blueprints; otherwise it could use `T[]`.

Reply:
It's a mix :( 
But we have tried to use `T[]` when possible.